kubernetes-templates/05_cvat_proxy_service.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cvat-proxy-service
+  namespace: cvat
+  labels:
+    app: cvat-app
+    tier: proxy
+spec:
+  type: NodePort
+  selector:
+      app: cvat-app
+      tier: proxy
+  ports:
+    - port: 80
+      targetPort: 80
+      protocol: TCP
```

Comment:
Do you think reasonable to specify a fixed nodePort as well? Also I think the mention in Readme.md on how to access the deployed CVAT might be helpful.

Reply:
Since the service is of type nodePort, i am not sure

I added the following to the section to the README.md in 34467324

### Expose the deployment
The service `cvat-proxy-service` is the accesspoint to the deployment.
In order to expose this resource an ingress might be handy [kubernetes ingress documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/).

For debugging puposes it is usefull to forward this service to a port on your localhost.
In the following example `8080` will be used for this purpose [localhost:8080](http://localhost:8080).

```bash
kubectl port-forward service/cvat-proxy-service -n cvat 8080:80
```

**Hint:**
If you are developing locally it might be usefull to replace `{MY_SERVER_URL_COM}` with `localhost`,
such that `/etc/hosts` does not need to override the DNS.

Reply:
Does this cover your request? Since the `cvat-proxy-service` is already of type NodePort, i am not sure what further change you are suggesting.

Please resolve this conversation if it covers your request.